### DISCUSSION TITLE
RDE Meetup Presentation on a New Meow Feature

### DISCUSSION THREAD

**Discussion by jgarte** [meow-edit/meow#464] [Show and tell]
Hi Meow users,

I just wanted to share a presentation I gave on meow for the third [rde](https://trop.in/rde/) meetup today. It was live streamed on [tropin](https://trop.in/)'s yt channel:

https://www.youtube.com/watch?v=kkwZw4DV8TU

RDE is an up and coming distribution of GNU Guix:

https://trop.in/rde/

The talk also presented a first prototype for [feature-emacs-meow](https://git.sr.ht/~whereiseveryone/confetti/tree/s/item/confetti/features/meow.scm), which is a [Guile Scheme](https://www.gnu.org/software/guile/) function that produces what's called a [feature](https://trop.in/rde/manual#Features) in the RDE system.

**Comment by DogLooksGood** [comment]
Just watched the presentation, it's amazing, thank you!
